src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classname from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo: React.FC<{ todo: Todo; }> = ({ todo }) => (
+  <div
+    className={classname('todo-status', 'todo', {
+      'todo-status--completed': todo.completed,
+    })}
+    key={todo.id}
```

Comment:
```suggestion
    key={todo.id}
```
Це має бути в методі map рівнем вище